docs/store/datastore.md

```diff
@@ -167,6 +167,82 @@ redis_profile = project.get_datastore_profile("my_profile")
 local_redis_profile = DatastoreProfileRedis(redis_profile.name, redis_profile.endpoint_url, username="mylocaluser", password="mylocalpassword")
 register_temporary_client_datastore_profile(local_redis_profile)
 ```
+```{admonition} Note
+Datastore profile does not support: v3io (datastore, or source/target), snowflake source, DBFS for spark runtimes, Dask runtime.
+```
+
+
+### Azure data store profile
+```
+profile = DatastoreProfileAzureBlob(name="test_profile",connection_string=connection_string)
+ParquetTarget(path="ds://test_profile/az_blob/path/to/parquet.pq")
```

Comment:
I don't think examples should contain the word "test" in them. It can be just a placeholder instead